### PR TITLE
Adds a guide on "How to update WCAG 2.1 translations"

### PR DESCRIPTION
**Pull request by remibetin**
This PR provides guidance for updating existing WCAG 2.1. translations.

It also adds a list of checks to perform before submitting a new WCAG translation to W3C.

Previews:
- (Updated) [How to Translate WCAG 2](https://deploy-preview-916--wai-website.netlify.app/about/translating/wcag/)
- (New) [How to update WCAG 2.1 translations](https://deploy-preview-916--wai-website.netlify.app/about/translating/wcag/wcag21-update/)

### CODE REVIEW

File: pages/about/translating/wcag.md [hunk 1]
```diff
@@ -131,7 +132,27 @@ Download [`fixup.js` {% include_cached external.html %}](https://www.w3.org/scr
...
-## Step 5: Deliver the final files to W3C {#deliver-files}
+## Step 5: Check your translation thoroughly {#check}
+
+Please properly check your files before submission to W3C.
+
+In particular, make sure that:
+- All links are valid. You can use [W3C Link Checker](https://validator.w3.org/checklink)
+- HTML is valid. You can use [W3C Markup Validation Service](https://validator.w3.org/)
+- CSS is valid. You can use [W3C CSS Validation Service](https://jigsaw.w3.org/css-validator/)
+- Pages are internationalisation-friendly. You can use [W3C Internationalization Checker](https://validator.w3.org/i18n-checker/)
```
Comment (shawna-slh) [inline pages/about/translating/wcag.md:143]: internationalisation -> internationalization

:)

File: pages/about/translating/wcag.md [hunk 2]
```diff
@@ -131,7 +132,27 @@ Download [`fixup.js` {% include_cached external.html %}](https://www.w3.org/scr
...
+- All relevant content has been properly translated
+- All `id`s are in English (= they have not been translated).
```
Comment (shawna-slh) [inline pages/about/translating/wcag.md:145]: how about:
> (= they have not been translated)

->
; that is, they have not been translated.

File: pages/about/translating/wcag.md [hunk 3]
```diff
@@ -131,7 +132,27 @@ Download [`fixup.js` {% include_cached external.html %}](https://www.w3.org/scr
...
+- `lang` and `xml:lang` attributes of the `<html>` tag are correct
+- `hreflang` attribute has been added to links when relevant
+- `lang` attribute has been used when relevant (especially when some text has been kept in English)
+- The links/anchors of the original and translated versions match.
+- “Jump to table of contents”, “Collapse sidebar” and “Hid sidebar” have been translated.
```
Comment (shawna-slh) [inline pages/about/translating/wcag.md:150]: Hid sidebar -> Hide sidebar

File: pages/about/translating/wcag.md (hunk 2 above)
Comment (shawna-slh) [reply pages/about/translating/wcag.md:145]: hey @remibetin For things that they should not do like translate IDs and add custom CSS, are those clearly explained early in the process? (We don't want them to have to undo that work!)

File: pages/about/translating/wcag.md (hunk 1 above)
Comment (remibetin) [reply pages/about/translating/wcag.md:143]: Fixed, thanks for spotting this!

File: pages/about/translating/wcag.md (hunk 2 above)
Comment (remibetin) [reply pages/about/translating/wcag.md]: > ; that is, they have not been translated.

Fixed.

> For things that they should not do like translate IDs and add custom CSS, are those clearly explained early in the process? (We don't want them to have to undo that work!)

I have added further indication.

File: pages/about/translating/wcag.md (hunk 3 above)
Comment (remibetin) [reply pages/about/translating/wcag.md]: Fixed

File: pages/about/translating/wcag.md [hunk 4]
```diff
@@ -131,7 +132,27 @@ Download [`fixup.js` {% include_cached external.html %}](https://www.w3.org/scr
...
+- There is no code showing in the rendered page.
+- No custom CSS has been added.
+- (Authorized translation) An errata has been created and is hosted on the Lead translating organization (LTO)'s website.
```
Comment (shawna-slh) [inline pages/about/translating/wcag.md:156]: I think capitalize Lead Translating Organization

**Review by shawna-slh — APPROVED** [review]
Thanks!

**Pull request merged**